polkadot/xcm/xcm-runtime-api/src/conversions.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Contains runtime APIs for useful conversions, such as between XCM `Location` and `AccountId`.
```

Comment:
There already exists multiple runtime apis for XCM. We should merge them all into one crate?

Reply:
yes, exactly, that is the open question from description, and also Adrian above is ok with that,
so I will merge them

Reply:
fixed here: https://github.com/paritytech/polkadot-sdk/pull/4857/commits/ab63f0dd2fc713c7ff45e4ecb58db076a223b600